kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/util/IwListChannelTool.java

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Sterwen-Technology
+ ******************************************************************************/
+package org.eclipse.kura.linux.net.util;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.kura.KuraErrorCode;
+import org.eclipse.kura.KuraException;
+import org.eclipse.kura.executor.Command;
+import org.eclipse.kura.executor.CommandExecutorService;
+import org.eclipse.kura.executor.CommandStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.eclipse.kura.net.wifi.WifiChannel;
+
+public class IwListChannelTool extends IwCapabilityTool {
```

Comment:
Since we are not using IwList but Iw this class still needs to extend IwCapabilityTool? Or those methods could be added to the extended class?